src/effects/VST3/VST3Effect.cpp

```diff
@@ -365,32 +364,32 @@ bool VST3Effect::SaveSettings(
 }
 
 bool VST3Effect::LoadSettings(
-   const CommandParameters & parms, EffectSettings &settings) const
+   const CommandParameters& parms, EffectSettings& settings) const
 {
    using namespace Steinberg;
 
-   if (mComponentHandler == nullptr)
-      return false;
+   VST3EffectSettings ownSettings;
```

Comment:
Could write: `auto &ownSettings = GetSettings(settings); ownSettings.clear();` and eliminate the assignment a the end

Reply:
good point, ok